src/quantuminspire/calibration_viewer.py

```diff
@@ -0,0 +1,130 @@
+""" Quantum Inspire SDK
+
+Copyright 2018 QuTech Delft
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from typing import Any, Dict
+
+import json
+import numpy as np
+
+try:
+    from rich import print as rprint
+except ImportError:
+    rprint = print
+
+
+class CalibrationViewer:
+
+    def __init__(self, calibration_dictionary: Dict[str, Any]) -> None:
+        """
+        Object to view and nicely print backend calibration data returned by the API.
+
+        :param calibration_dictionary: The calibration dictionary to be used by viewer.
+        """
+        self._calibration = calibration_dictionary
+        return
+
+    @property
+    def calibration(self) -> Dict[str, Any]:
+        """The property containing the calibration dictionary to be used."""
+        return self._calibration
+
+    @property
+    def timestamp(self) -> str:
+        """Timestamp of the last calibration of the current calibration dictionary."""
+        return str(self.calibration['parameters']['system']['last_calibration_date']['value'])
+
+    def show_system_parameters(self) -> None:
+        """Print system calibration parameters of the calibration dictionary."""
+        system = self.calibration['parameters']['system']
+        rprint(f'[green]System[/green]:')
+
+        for key, item in system.items():
+            if isinstance(item, dict):
+                if np.all([k in item for k in ['unit', 'value']]):
+                    rprint(f"\t{key}: {item['value']} [{item['unit']}]")
+                else:
+                    try:
+                        rprint(f"\t{key}: calibration at {item['timestamp']}")
+                    except KeyError:
+                        rprint(f"Key 'timestamp' not in item {item}.")
```

Comment:
1. the method is called: show_system_parameters so I expect the system structure to be printed. But what I get is only a timestamp?
2. when timestamp is not in the structure we get lines with key 'timestamp' not in messages. Why not just print item in this case as I would expect from the method name (and as done in show_qubit_parameters)